CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 - PR #3442 Add Bool-index + Multi column + DataFrame support for set-item
 - PR #3172 Define and implement new fill/repeat/copy_range APIs
 - PR #3497 Add DataFrame.drop(..., inplace=False) argument
+- PR #3438 first working attempt to use scalar_device_view in search
```

Comment:
Please move this entry to the `Improvements` section, this is not a new feature.

```suggestion
- PR #3438 use scalar_device_view in search
```